src/test/java/com/cloudbees/jenkins/plugins/bitbucket/SCMNavigatorTest.java

```diff
@@ -85,26 +86,29 @@ public SCMSourceObserverImpl(TaskListener listener) {
             this.listener = listener;
         }
 
+        @NonNull
```

Comment:
`@NonNull` and then `return null`, something is wrong here (maybe not affecting to tests as a lot of code is mocked)